ophyd/epics_motor.py

```diff
@@ -18,11 +18,17 @@
 from .positioner import PositionerBase
 from .device import (Device, Component as Cpt)
 from .status import wait as status_wait
+from enum import Enum
 
 
 logger = logging.getLogger(__name__)
 
 
+class HomeEnum(str, Enum):
```

Comment:
Like `enum.IntEnum`, it allows you to compare the enum value with the builtin-typed value. For example, `HomeEnum.forward == 'forward'` == `True`. Without this, the comparison will fail.